### ISSUE TITLE
code should be <pre>'d

### ISSUE THREAD

**Issue by joscha** [bugcrowd/test-summary-buildkite-plugin#24]
When running a test that outputs things like:
![web__build_editor__20276](https://user-images.githubusercontent.com/188038/43118054-58fad80c-8f53-11e8-8b54-9655f8819c1e.png)

for a test failure, this output gets mangled in the summary:
![web__build_editor__20276](https://user-images.githubusercontent.com/188038/43118057-63b3d1f4-8f53-11e8-89ff-b820dd48cc59.png)

I suspect a missing `<code><pre class="diff"></pre></code>` somewhere.

**Comment by tessereth** [comment]
Hm, that's interesting. Any chance you can give a sample junit/tap output file so I can test it out?

**Comment by joscha** [comment]
That's the `junit.xml` generated by `jest`:

```xml
<testsuites name="jest editor tests" tests="8772" failures="1" time="174.01">
  <testsuite name="HeaderStore" errors="0" failures="1" skipped="0" timestamp="2018-07-25T03:01:53" time="0.678" tests="7">
    <testcase classname="Header › matches snapshot" name="Header › matches snapshot" time="0.03">
      <failure>Error: expect(value).toMatchSnapshot()

Received value does not match stored snapshot &quot;Header matches snapshot 1&quot;.

- Snapshot
+ Received

@@ -3,11 +3,11 @@
  &gt;
    &lt;nav
      class=&quot;startGroup&quot;
    &gt;
      &lt;a
-       class=&quot;base anchor button booo homeLink&quot;
+       class=&quot;base anchor button linkButton homeLink&quot;
        href=&quot;/&quot;
      &gt;
        &lt;span
          class=&quot;text&quot;
        &gt;
    at Object.toMatchRenderedSnapshot (/build/web/tools/jest/matchers/to_match_rendered_snapshot.ts:9:25)
    at Object.throwingMatcher [as toMatchRenderedSnapshot] (/build/web/node_modules/expect/build/index.js:316:33)
    at Object.it (/build/web/src/pages/editor/header/tests/header.tests.tsx:76:9)
    at Object.asyncFn (/build/web/node_modules/jest-jasmine2/build/jasmine_async.js:108:37)
    at resolve (/build/web/node_modules/jest-jasmine2/build/queue_runner.js:56:12)
    at new Promise (&lt;anonymous&gt;)
    at mapper (/build/web/node_modules/jest-jasmine2/build/queue_runner.js:43:19)
    at promise.then (/build/web/node_modules/jest-jasmine2/build/queue_runner.js:87:41)</failure>
    </testcase>
  </testsuite>
</testsuites>
```

**Comment by joscha** [comment]
looking at the haml files it might not be a missing pre, but some other logic in the BK annotations that treats the `-` and `+` at the beginning of the line as bullets. And then additionally I think we would want to decode HTML entities in failures.

**Comment by tessereth** [comment]
Should be fixed in v1.5.0. Thanks for the bug report!
